storage/innobase/handler/ha_innodb.cc

```diff
@@ -11958,6 +11983,19 @@ create_table_info_t::create_options_are_invalid()
 	return(ret);
 }
 
+void
+ha_innobase::adjust_create_info_for_frm(
+	HA_CREATE_INFO*	create_info)
+{
+		if (create_info->encrypt_type.length == 0
+			&& create_info->encrypt_type.str == NULL
+			&& srv_encrypt_tables != 0
+			) {
+				static const LEX_STRING yes = { C_STRING_WITH_LEN("Y") };
```

Comment:
Function-level static variables/constants are not potentially thread-safe in c++98 (even for POD data types). Please, move to the global level or class level
`/*static*/ const LEX_STRING ha_innobase::yes = { C_STRING_WITH_LEN("Y") };`